tests/Aspire.Playground.Tests/AppHostTests.cs

```diff
@@ -285,16 +318,51 @@ public static IList<TestEndpoints> GetAllTestEndpoints()
                     new ("basketservice", "Application started"),
                     new ("postgres", "database system is ready to accept connections"),
                 ]),
+            // Test endpoints disabled for AppHostRunsCleanlyForAppsWithNoOtherTests as they have specific tests
+            new TestEndpoints("WithDockerfile.AppHost", []),
+            new TestEndpoints("KafkaBasic.AppHost", []),
+            new TestEndpoints("AzureFunctionsEndToEnd.AppHost", [])
```

Comment:
It feels like this should be a separate "exclusion" list or something. Putting these empty ones in here feels wrong.

Reply:
Or maybe then the `AppHostRunsCleanly*` test should get an explicit list, which would be `[]` right now. And if we add more apps which we want to at least build-run check, then we can add those explicitly.

Reply:
I dropped the test entirely as it has no real purpose now. Any apps that we are testing are either tested with `TestEndpointsReturnOk`, or with more specific tests.
The only use left for the `AppHostRunsCleanly` was for apps that we just want to build+run, but even there we don't really check for errors, so it isn't useful.